src/valkey-cli.c

```diff
@@ -140,12 +140,14 @@
 #define CLUSTER_MANAGER_CMD_FLAG_FIX_WITH_UNREACHABLE_PRIMARIES 1 << 10
 #define CLUSTER_MANAGER_CMD_FLAG_PRIMARIES_ONLY 1 << 11
 #define CLUSTER_MANAGER_CMD_FLAG_REPLICAS_ONLY 1 << 12
+#define CLUSTER_MANAGER_CMD_FLAG_USE_ATOMIC_SLOT_MIGRATION 1 << 13
```

Comment:
The better convention for macros is to wrap it in brackets so it can be negated and similar without worrying. We could do if for all these macros.

```suggestion
#define CLUSTER_MANAGER_CMD_FLAG_USE_ATOMIC_SLOT_MIGRATION (1 << 13)
```

Reply:
Makes sense, done now